WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -21,6 +22,37 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 		{
 		}
 
+		#region blockchain
+
+		public async Task<IEnumerable<string>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)
```

Comment:
Did you not succeed to use FilterModel here?

Reply:
Returning the correct type now.